index.html

```diff
@@ -21,7 +21,9 @@
   <div class="container">
     <div class="left">
       <h1 class="title">WELCOME TO HEALTHY DEVELOPER</h1>
-      <button class="button-81" role="button">Explore</button>
+      <a href="/explore.html">
```

Comment:
https://github.com/asdsyd/Healthy-developer/blob/main/explore.html
Has to be used. Nvm I'll Fix it.
Thank you for contribution ❤️

Reply:
> https://github.com/asdsyd/Healthy-developer/blob/main/explore.html Has to be used. Nvm I'll Fix it. Thank you for contribution ❤️

Sure! Thank you.